test/REQUIRE

```diff
@@ -1 +1 @@
-ValidatedNumerics 0.5.0
+TaylorSeries 0.4.0
```

Comment:
I put TaylorSeries there because it simplifies the notation for the jet transport tests